Habitica/src/main/java/com/habitrpg/android/habitica/ContentCache.java

```diff
@@ -19,10 +19,17 @@
 
 public class ContentCache {
     private ApiService apiService;
+    private String language;
+
 
     public ContentCache(ApiService apiService) {
+        this.apiService = apiService;
+        this.language = "en";
+    }
 
+    public ContentCache(ApiService apiService, String language) {
```

Comment:
I feel like this should be refactored a bit. Right now, the APIHelper has knowledge of the language, but to create a contentCache, you then have to take that language and then also store it in the contentCache.
I think it makes more sense, to give the contentCache an instance of the APIHelper and then have a new getContent() method on the APIHelper. That calls the getContent() method on the apiService using the correct language. That way you can request content without having to pass in the language every time.